test/upgrade/service_postupgrade_test.go

```diff
@@ -62,6 +74,30 @@ func TestBYORevisionPostUpgrade(t *testing.T) {
 	}
 }
 
+func configUpdated(t *testing.T, clients *test.Clients, serviceName string) bool {
```

Comment:
I would recommend taking the generation as a value to the function to make it more generically useful.

i.e. `ConfigHasGeneration(1, ...)`